app/imports/api/users/server/createUser.js

```diff
@@ -40,10 +40,27 @@ const setAdminOnFirstUser = (options, user) => {
     return user;
 };
 
+const getPublicEmail = (user) => {
+    const githubEmail = [{
+        address: _.get(user, 'services.github.email', undefined),
```

Comment:
Please don't add `[{ address: undefined, verified: true }]` email address should be an email, not undefined ;)